app/e2e/common/pages/playwright-landing-page.ts

```diff
@@ -0,0 +1,158 @@
+// playwright-landing-page.ts
+import { expect, Locator, Page } from '@playwright/test';
+
+interface RegisterForm {
+    email: string;
+    password: string;
+    firstName: string;
+    lastName: string;
+}
+
+interface LoginForm {
+    email: string;
+    password: string;
+}
+
+export class PlaywrightLandingPage {
+    readonly page: Page;
+    // Buttons
+    readonly appBarHamburgerButton: Locator;
+    readonly appBarRegisterButton: Locator;
+    readonly appBarLoginButton: Locator;
+    readonly landingButton: Locator;
+    readonly nextButtons: Locator;
+    readonly backButtons: Locator;
+    // Logos/Graphics
+    readonly appBarPrytaneumLogo: Locator;
+    readonly prytanumTextLogo: Locator;
+    readonly prytaneumLogoSubheader: Locator;
+    readonly landingGraphic: Locator;
+    readonly bottomPrytaneumLogo: Locator;
+    readonly DemocracyFundLogo: Locator;
+    readonly UCRTecdLogo: Locator;
```

Comment:
It seems as though the scope of this is a bit much. Are all of these unique to the page? For example, I think we could have a generalized `Nav` POM potentially. Otherwise, `Nav` buttons will exist in every single POM since we pretty much always have a nav.

In general, I think this should probably only encapsulate things that are unique to the page or have a unique behavior on this particular page.

Reply:
Good point, I'm open to discussing strategies to improve this.